tests/models/vivit/test_modeling_vivit.py

```diff
@@ -359,12 +359,12 @@ def test_inference_interpolate_pos_encoding(self):
         # allowing to interpolate the pre-trained position embeddings in order to use
         # the model on higher resolutions. The DINO model by Facebook AI leverages this
         # to visualize self-attention on higher resolution images.
-        model = VivitModel.from_pretrained("google/vivit-b-16x2").to(torch_device)
+        model = VivitModel.from_pretrained("google/vivit-b-16x2-kinetics400").to(torch_device)
 
-        image_processor = VivitImageProcessor.from_pretrained("google/vivit-b-16x2")
+        image_processor = VivitImageProcessor.from_pretrained("google/vivit-b-16x2-kinetics400")
```

Comment:
Why change the checkpoint and the crop_size in the test?

Reply:
@amyeroberts 
- Changing the checkpoint was a test to see if that fixes the test failure in `PR Slow CI` that you triggered. (The one I mentioned above)
- different `crop_size`(s) leads to an error during the calling of interpolation method for example when the `crop_size` was `crop_size={"height": 480, "width": 480}` the following error occurs:
```
# add positional encoding to each token
        if interpolate_pos_encoding:
>           embeddings = embeddings + self.interpolate_pos_encoding(embeddings, height, width)
E           RuntimeError: The size of tensor a (14401) must match the size of tensor b (901) at non-singleton dimension 1
```
same happens with some other crop sizes as well. But the error doesn't occur for `crop_size` like 232 or 228

Reply:
OK, thanks for explaining. The error shouldn't be triggered for the default `crop_size` value (no interpolation should happen) but if it works for these none default values then it's all good :)

Reply:
The default `crop_size` value is 224, right?!, in all the image processing files. The error doesn't occur for that value tho. This value is only given in the test file so that the error doesn't occur & for the sake of testing a value apart from the default one.